app/status.py

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/python2.7
```

Comment:
If you want to focus on the UI refresh only in this changeset, it's also okay to leave the code as is with a TODO to cleanup duplicate code in this file at a later iteration and file an issue in github so we don't forget it.